src/index.html

```diff
@@ -5,8 +5,290 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products">
+      <article class="product_card" data-qa="card">
+        <img
+          class="product_photo"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          >
+
+        <h2 class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="product_code">Product code: 195434</p>
+
+        <div class="note">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="reviews">Reviews: 5</div>
+        </div>
+
+        <p class="price">
+          <span class="price_name">Price:</span>
+          <span class="price_number">$2,199</span>
+        </p>
+
+        <a class="button_buy" data-qa="card-hover">BUY</a>
+      </article>
+      <article class="product_card" data-qa="card">
+        <img
+          class="product_photo"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          >
+
+        <h2 class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="product_code">Product code: 195434</p>
+
+        <div class="note">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="reviews">Reviews: 5</div>
+        </div>
+
+        <p class="price">
+          <span class="price_name">Price:</span>
+          <span class="price_number">$2,199</span>
+        </p>
+
+        <a class="button_buy" data-qa="card-hover">BUY</a>
+      </article>
+      <article class="product_card" data-qa="card">
+        <img
+          class="product_photo"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          >
```

Comment:
Fix code style everywhere
```suggestion
        <img
          class="product_photo"
          src="images/imac.jpeg"
          alt="APPLE A1419 iMac 27"
        >
```